commit 7ea4e095ad6e5aa8f12452609a2e70f94c5e570a (from 49ed4696eaf0c14c273cbc2ec4733d058383ba7a)
Merge: 49ed469 bcbac69
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Mar 23 22:43:11 2016 -0700

    Merge pull request #3220 from thinkyhead/rc_tori_spelling
    
    Patch minor spelling error in cn, en, gl language file headings

diff --git a/Marlin/language_cn.h b/Marlin/language_cn.h
index 8e82afd..a2c797a 100644
--- a/Marlin/language_cn.h
+++ b/Marlin/language_cn.h
@@ -2,7 +2,7 @@
  * Chinese
  *
  * LCD Menu Messages
- * Se also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
+ * See also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
  *
  */
 #ifndef LANGUAGE_CN_H
diff --git a/Marlin/language_en.h b/Marlin/language_en.h
index 271badb..819ae99 100644
--- a/Marlin/language_en.h
+++ b/Marlin/language_en.h
@@ -2,7 +2,7 @@
  * English
  *
  * LCD Menu Messages
- * Se also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
+ * See also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
  *
  */
 #ifndef LANGUAGE_EN_H
diff --git a/Marlin/language_gl.h b/Marlin/language_gl.h
index 12806db..fc9c8c3 100644
--- a/Marlin/language_gl.h
+++ b/Marlin/language_gl.h
@@ -2,7 +2,7 @@
  * Galician language (ISO "gl")
  *
  * LCD Menu Messages
- * Se also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
+ * See also https://github.com/MarlinFirmware/Marlin/wiki/LCD-Language
  *
  */
 #ifndef LANGUAGE_GL_H

commit 7ea4e095ad6e5aa8f12452609a2e70f94c5e570a (from bcbac698f26f51aaac16bc11572fd45b5ef74c2b)
Merge: 49ed469 bcbac69
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Mar 23 22:43:11 2016 -0700

    Merge pull request #3220 from thinkyhead/rc_tori_spelling
    
    Patch minor spelling error in cn, en, gl language file headings

diff --git a/Marlin/pins.h b/Marlin/pins.h
index f841cd7..6d97df2 100644
--- a/Marlin/pins.h
+++ b/Marlin/pins.h
@@ -219,6 +219,7 @@
 #endif
 
 #if ENABLED(DISABLE_YMAX_ENDSTOP)
+  #undef Y_MAX_PIN
   #define Y_MAX_PIN          -1
 #endif
 

